extra/datadog.sh

```diff
@@ -26,6 +26,24 @@ cp $DATADOG_CONF.example $DATADOG_CONF
 sed -i -e"s|^.*confd_path:.*$|confd_path: $DD_CONF_DIR/conf.d|" $DATADOG_CONF
 sed -i -e"s|^.*additional_checksd:.*$|additional_checksd: $DD_DIR/checks.d|" $DATADOG_CONF
 
+# Include application's datadog configs
+APP_DATADOG_CONF_DIR="/app/datadog/conf.d"
+APP_DATADOG_CHECKS_DIR="/app/datadog/checks.d"
+
+for file in "$APP_DATADOG_CONF_DIR"/*.yaml; do
+  filename=$(basename -- "$file")
+  filename="${filename%.*}"
+  mkdir -p "$DD_CONF_DIR/conf.d/${filename}.d"
+  cp $file "$DD_CONF_DIR/conf.d/${filename}.d/conf.yml"
+done
+
+for file in "$APP_DATADOG_CHECKS_DIR"/*.yaml; do
```

Comment:
Sorry, I should have been a bit more clear. I was just asking for the change on line 30.

The `checks.d` dir contains python files that correspond to the yaml files and they'll also need to be added to the python path (logic around line 82). Let's keep this commit short and delete line 31 and this section. I can merge the config file part and we can work on adding logic bits in a separate PR.

Thanks again for all your work on this!